src/main/java/io/penguinstats/task/UpdateDropMatrixTask.java

```diff
@@ -16,11 +17,15 @@
 	@Autowired
 	private ItemDropService itemDropService;
 
-	@Scheduled(fixedRate = 600000)
+	// TODO: remove initialDelay
```

Comment:
when?

Reply:
This is for testing purpose, i don't want this job to run when I start the app every time.